Makefile

```diff
@@ -68,10 +68,10 @@ coverage: clean
 	$(BROWSER) htmlcov/index.html
 
 isort_check: ## check that isort has been run
-	isort --check-only -rc enterprise_access/
+	isort --check-only enterprise_access/
 
 isort: ## run isort to sort imports in all Python files
-	isort --recursive --atomic enterprise_access/
```

Comment:
why no recursive?

Reply:
--recursive is a deprecated and ignored flag, and now is the default.